<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExistingPoolProvisionDialog</class>
 <widget class="QWidget" name="ExistingPoolProvisionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_pool">
     <item>
      <widget class="QLabel" name="label_pool">
       <property name="text">
        <string>Pool</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pool"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_pool"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_pool_select">
     <property name="text">
      <string>Confirm Pool Selection</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_input_token">
     <item>
      <widget class="QLabel" name="label_input_token">
       <property name="text">
        <string>Input Token</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_input_token"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_input_token">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Input token quantity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_provide">
     <property name="text">
      <string>Provide</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
